### DISCUSSION TITLE
3D Earth and Celestial Bodies (planet3D)

### DISCUSSION THREAD

**Discussion by tkoyama010** [pyvista/pyvista#1501] [Ideas]
We can plot the 3D Earth and Celestial Bodies manually in pyvista.

https://docs.pyvista.org/user-guide/jupyter/panel.html#examples-and-usage

But it is better to add dedicated function to plot 3D Earth and Celestial Bodies like `planet3D` in MATLAB.
https://jp.mathworks.com/matlabcentral/fileexchange/86483-3d-earth-and-celestial-bodies-planet3d

![planet3D](https://jp.mathworks.com/matlabcentral/mlc-downloads/downloads/2b90c6ac-512c-472d-a051-9b0d6aa91ddd/43f2f993-a4c4-4d6f-9481-a013608cbbfb/images/1611827475.jpg)

**Comment by akaszynski** [comment]
I'd love to have a module that includes examples of earth and other planets!

**Comment by tkoyama010** [comment]
Also see https://blogs.mathworks.com/pick/2021/05/31/visualizing-earth-and-celestial-bodies

![visualizing-earth-and-celestial-bodies](https://blogs.mathworks.com/pick/files/potw_3D_planets_1-3.gif)

**Reply by tkoyama010** [reply]
Script will be..
```python
from pyvista import examples

mercury = examples.load_mercury()
venus = examples.load_venus()
globe = examples.load_globe()
mars = examples.load_mars()
jupiter = examples.load_jupiter()
saturn = examples.saturn() # This is most difficult!
```
`load_globe` is already exist :)

**Reply by adeak** [reply]
The Moon orbits in the other direction, though :) Same sign of the angular momentum vector as Earth's spin and orbital moment. (I know you didn't create that animation.)

**Reply by tkoyama010** [reply]
:laughing: The world can see the dark side of the moon!
